test/test.rapptor.js

```diff
@@ -62,11 +33,13 @@ tap.test('response method ', async t => {
   const response1 = await handler({
     path: '/api',
     method: 'get',
+    headers: {},
```

Comment:
Why was this added?

Reply:
A bunch of the tests break because we're now logging the user-agent and referer headers, this just fixes those tests so they have a headers field and don't crash.